inc/admin.php

```diff
@@ -831,8 +856,17 @@ function render_form( $widget, $instance = array(), $raw = false, $widget_number
 		$the_widget->number = $widget_number;
 
 		ob_start();
+		if ( $this->is_js_widget( $the_widget ) ) {
+		?><div class="widget-content"><?php
+		}
 		$return = $the_widget->form( $instance );
 		do_action_ref_array( 'in_widget_form', array( &$the_widget, &$return, $instance ) );
+		if ( $this->is_js_widget( $the_widget ) ) {
+		?>
```

Comment:
Would be a little easier to follow these blocks in the `if` statements were indented one level.